modules/phase_field/include/postprocessors/GrainBoundaryArea.h

```diff
@@ -0,0 +1,41 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef GRAINBOUNDARYAREA_H
+#define GRAINBOUNDARYAREA_H
+
+#include "ElementIntegralPostprocessor.h"
+
+class GrainBoundaryArea;
+
+template<>
+InputParameters validParams<GrainBoundaryArea>();
+
+/**
+ * Calculate total grain boundary area.
```

Comment:
how about mentioning here that it will give you GB length in 2D and GB area in 3D